ChineseFilter/sections/general_url.txt

```diff
@@ -1,4 +1,10 @@
 !
+! This section contains the list of the generic rules against ads.
+!
+! Good: .org/ads/
+! Bad: .org/ads/$domain=example.org (for instance, should be in general_extensions.txt)
```

Comment:
Why `general_extensions.txt`? This is a domain-specific rule.

Reply:
Indeed.